app/services/end_of_cycle_timetable.rb

```diff
@@ -60,12 +60,10 @@ def self.date(name)
   end
 
   def self.current_cycle_schedule
-    return :real unless HostingEnvironment.test_environment? || HostingEnvironment.sandbox_mode?
-    return :today_is_between_cycles if FeatureFlag.active?(:simulate_time_between_cycles)
-    return :today_is_mid_cycle if FeatureFlag.active?(:simulate_time_mid_cycle)
-    return :today_applications_are_unavailable_to_stopped_courses if FeatureFlag.active?(:simulate_stop_applications_to_unavailable_course_options)
+    # Make sure this setting only has effect on non-production environments
+    return :real if HostingEnvironment.production?
```

Comment:
Does this force `sandbox` to always use `:real`? (`sandbox` doesn't seem to be a hosting environment so I'm not sure). If so that will disrupt vendor/provider testing.

Reply:
No, but we should make sure to turn on `today_is_mid_cycle` on sandbox as soon as this is deployed.